rules/style.css

```diff
@@ -143,20 +156,32 @@ body {
   background-color: var(--main-background-color);
 }
 
-main.main-content, div.footnotes {
+main.main-content,
+div.footnotes {
   background-color: var(--text-background-color);
 }
 
-nav.TOC, nav.TOC a, nav.TOC a:visited {
+nav.TOC,
+nav.TOC a,
+nav.TOC a:visited {
   background-color: var(--accent-background-color);
   color: var(--main-background-color);
 }
 
-nav.TOC span:hover, nav.TOC span:hover *{
+nav.TOC span:hover,
+nav.TOC span:hover * {
   background-color: var(--accent-background-color);
 }
 
-figure.texsource, figure.shellcommand, figure.htmlsource, figure.luasource, figure.textsource{
+figure.texsource,
+figure.shellcommand,
+figure.htmlsource,
+figure.luasource,
+figure.textsource {
   border: 1px solid var(--main-background-color);
   background-color: var(--accent-background-color);
 }
+
+.table {
+  overflow-x: scroll;
+}
```

Comment:
```suggestion
}

```
small style tweak

Reply:
This newline here is the 'small fix'

Reply:
Oh I see, but you edited already. I have enabled "Edits by maintainers" so it should be done already